### ISSUE TITLE
Update ./bin/setup

### ISSUE THREAD

**Issue by rlgreen91** [EBWiki/EBWiki#3383]
Update the `./bin/setup` file to make a copy of `config/database.example.yml` and `.env.example` when run.  Additionally, add functionality to the file to check if Postgres, Redis, and Elasticsearch are installed on the system and running.

**Comment by kusumandaru** [comment]
i will look for these ticket

**Comment by rlgreen91** [comment]
@managedkaos Do you think we still need this at this point?

**Comment by managedkaos** [comment]
will take a look.  at the moment I don't think so... but I'm wondering if it will be useful for the errors we are seeing when running inside the container...

**Comment by managedkaos** [comment]
I don't think this is needed.  it may be useful for the full stack workflow but not so much for the docker workflow.

**Comment by rlgreen91** [comment]
@managedkaos Can you confirm that this is no longer needed?  If so, then please feel free to close the issue.

**Comment by managedkaos** [comment]
Will do this weekend
